tests/Symfony/Installer/Tests/IntegrationTest.php

```diff
@@ -106,6 +106,12 @@ public function provideSymfonyInstallationData()
                 '/.*Symfony 2\.5\.6 was successfully installed.*/',
                 '/Symfony version 2\.5\.6 - app\/dev\/debug/',
             ),
+
+            array(
+                '2.7.0-BETA1',
+                '/.*Symfony 2\.7\.0\-BETA1 was successfully installed.*/',
+                '/Symfony version 2\.7\-BETA1 - app\/dev\/debug/',
```

Comment:
I'm trying to do things fast ... but I do everything wrong :(  Thanks for reviweing this!